docs/make_projection.py

```diff
@@ -78,31 +63,27 @@ def projection_rst(projection_cls):
     table.write('Cartopy projection list\n')
     table.write('=======================\n\n\n')
 
-    prj_class_sorter = lambda cls: (PRJ_SORT_ORDER.get(cls.__name__, []), cls.__name__)
+    prj_class_sorter = lambda cls: (PRJ_SORT_ORDER.get(cls.__name__, []),
+                                    cls.__name__)
     for prj in sorted(find_projections(), key=prj_class_sorter):
         name = prj.__name__
-#        print prj in SPECIAL_CASES, prj in all_projections_in_groups, prj
-
-        # put the class documentation on the left, and a sidebar on the right.
-
-        aspect = (np.diff(prj().x_limits) / np.diff(prj().y_limits))[0]
-        width = 3 * aspect
-        if width == int(width):
-            width = int(width)
 
         table.write(name + '\n')
         table.write('-' * len(name) + '\n\n')
 
         table.write('.. autoclass:: cartopy.crs.%s\n' % name)
 
-#        table.write('Ipsum lorum....')
-
-#        table.write("""\n\n
-#
-#.. sidebar:: Example
-#""")
-
-        for instance_creation_code in SPECIAL_CASES.get(prj, ['%s()' % name]):
+        for instance_args in SPECIAL_CASES.get(prj, [{}]):
+            prj_inst = prj(**instance_args)
+            aspect = (np.diff(prj_inst.x_limits) /
+                      np.diff(prj_inst.y_limits))[0]
+            width = 3 * aspect
+            if width == int(width):
+                width = int(width)
+
+            instance_params = ', '.join('{}={}'.format(k, v)
+                                        for k, v in instance_args.items())
```

Comment:
Sorry for being confusing, but this change is actually part of #505, which you've already merged.